index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Fred's Personal Portfolio</title>
+</head>
+<body>
+    <div class="header-container">
+        <header class="toolbar-section">
+            <nav class="nav-menu">
+                <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
+                <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
+            </nav>
+        </header>
+        <section class="headline-section">
```

Comment:
- [ ] [OPTIONAL] try to stick to design by using the same background as the Figma design 
- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164516351-57810e90-b39c-46de-81ed-017681907d52.png)
- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164516389-42217318-1972-4eb2-a582-d155a0f47448.png)